Models/Spectrometer.cs

```diff
@@ -181,6 +181,9 @@ public async Task<bool> initAsync(Dictionary<string, ICharacteristic> characteri
         logger.debug("Spectrometer.initAsync: generating pixel axis");
         generatePixelAxis();
 
+        // disable laserWarningDelay (testing)
+        laserWarningDelaySec = 0;
```

Comment:
I explicitly set laserWarningDelay to zero in the connection routine because we don't yet have a GUI element to control this (didn't take time to make one).